app/controllers/hotels_controller.rb

```diff
@@ -1,2 +1,47 @@
 class HotelsController < ApplicationController
+    before_action :check_admin, only: [:new, :edit]
```

Comment:
何をcheckするのか明確でないため、checkという名前のメソッドは実はアンチパターンになります。

具体的な名前のメソッドをつけましょう。

以下はいかがでしょう？

```suggestion
    before_action :deny_non_admin_users, only: [:new, :edit]
```

Reply:
ありがとうございます！
こちらの名前を使わせて頂きます🙇🏼‍♂️